tests/integration/tests/charm_tests/openstack_exporter.py

```diff
@@ -107,6 +109,10 @@ def test_configure_ssl_ca(self):
         results = model.run_on_leader(APP_NAME, command)
         self.assertEqual(int(results.get("Code", "-1")), 0)
 
+        # Enable cache again
+        model.set_application_config(APP_NAME, {"cache": "true"})
+        model.block_until_all_units_idle()
```

Comment:
I believe this should be done by clean up function, so it will be set to true even if tests failed.